### PR TITLE
Assmos tweaks/minor additions.

### PR DESCRIPTION
**Pull request by as334**
:cl: as334
add: Pluoxium can now be formed by irradiating tiles with CO2 in the air.
add: Rad collectors now steadily form Tritium at a slow pace.
balance: Nerfs fusion by making it slower, and produce radioactivity.
balance: Noblium formation now requires significantly more energy input.
balance: Tanks now melt if their temperature is above 1 Million Kelvin.
/:cl:

### CODE REVIEW

**Comment by nicbn** [comment]
Remove the N2O as NO2 catalyst please

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -355,3 +355,9 @@
 	if(wet_overlay)
 		cut_overlay(wet_overlay)
 
+
+/turf/open/rad_act(pulse_strength)
+	if (air.gases[/datum/gas/carbon_dioxide])
+		air.gases[/datum/gas/carbon_dioxide][MOLES]-=(pulse_strength/100)
```
Comment (kevinz000) [inline code/game/turfs/open.dm]: could possibly result in negatives.

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 2]
```diff
@@ -210,22 +211,23 @@
 	var/reaction_energy = THERMAL_ENERGY(air)
 	var/moles_impurities = air.total_moles()-(cached_gases[/datum/gas/plasma][MOLES]+cached_gases[/datum/gas/tritium][MOLES])
 
-	var/plasma_fused = (PLASMA_FUSED_COEFFICENT*catalyst_efficency)*(temperature/PLASMA_BINDING_ENERGY)*4
-	var/tritium_catalyzed = (CATALYST_COEFFICENT*catalyst_efficency)*(temperature/PLASMA_BINDING_ENERGY)
+	var/plasma_fused = (PLASMA_FUSED_COEFFICENT*catalyst_efficency)*(temperature/PLASMA_BINDING_ENERGY)/10
+	var/tritium_catalyzed = (CATALYST_COEFFICENT*catalyst_efficency)*(temperature/PLASMA_BINDING_ENERGY)/40
 	var/oxygen_added = tritium_catalyzed
-	var/waste_added = (plasma_fused-oxygen_added)-((air.total_moles()*air.heat_capacity())/PLASMA_BINDING_ENERGY)
+	var/waste_added = max((plasma_fused-oxygen_added)-((air.total_moles()*air.heat_capacity())/PLASMA_BINDING_ENERGY))
```
Comment (ninjanomnom) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: The max here is unused

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

File: code/modules/power/singularity/collector.dm [hunk 3]
```diff
@@ -37,6 +37,8 @@
 			eject()
 		else
 			loaded_tank.air_contents.gases[/datum/gas/plasma][MOLES] -= 0.001*drainratio
+			ASSERT_GAS(/datum/gas/tritium,loaded_tank.air_contents)
+			loaded_tank.air_contents.gases[/datum/gas/tritium][MOLES] += 0.001*drainratio
```
Comment (ninjanomnom) [inline code/modules/power/singularity/collector.dm]: oh god it perfectly converts all the plasma?

Also this is going to make collector spam a thing again, don't worry though I'll fix that in the next major balance pass.

File: code/game/turfs/open.dm (hunk 1 above)
Comment (ninjanomnom) [reply code/game/turfs/open.dm]: This makes co2 engine setups impossible as it'll convert all the co2 quite quickly

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 4]
```diff
@@ -210,22 +211,23 @@
...
 	cached_gases[/datum/gas/tritium][MOLES] = max(cached_gases[/datum/gas/tritium][MOLES] - tritium_catalyzed,0)
 	cached_gases[/datum/gas/oxygen][MOLES] += oxygen_added
-	cached_gases[/datum/gas/nitrogen][MOLES] += waste_added
 	cached_gases[/datum/gas/water_vapor][MOLES] += waste_added
 	cached_gases[/datum/gas/nitrous_oxide][MOLES] += waste_added
 	cached_gases[/datum/gas/nitryl][MOLES] += waste_added
-
+	cached_gases[/datum/gas/carbon_dioxide][MOLES] += waste_added
+	if (location)
+		radiation_pulse(location, 300)
```
Comment (ninjanomnom) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: shouldn't this scale with reaction energy?

**Review by ninjanomnom — DISMISSED** [review]

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 2 above)
Comment (duncathan) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: ^

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 4 above)
Comment (duncathan) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: whatever you think is best here ninja; radiation is your deal

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by duncathan** [comment]
@as334

**Comment by as334** [comment]
@duncathan

**Review by duncathan — APPROVED** [review]

**Pull request merged**